vpn-internal/src/main/java/com/duckduckgo/vpn/internal/feature/VpnInternalSettingsActivity.kt

```diff
@@ -106,13 +113,60 @@ class VpnInternalSettingsActivity : DuckDuckGoActivity() {
         setupBadHealthMonitoring()
         setupConfigSection()
         setupUiElementsState()
+        setupAppProtectionSection()
     }
 
     override fun onDestroy() {
         super.onDestroy()
         transparencyModeDebugReceiver?.let { it.unregister() }
     }
 
+    private fun setupAppProtectionSection() {
+        appTrackerRepository.getAppExclusionListFlow()
+            .combine(appTrackerRepository.getManualAppExclusionListFlow()) { exclusionList, manualList ->
+                val mappedExclusionList = exclusionList.map { it.packageId }
+                val mappedManualExclusionList = manualList.map { it.packageId }
+
+                val canProtect = mappedManualExclusionList.isEmpty() ||
+                    !mappedManualExclusionList.zip(mappedExclusionList).all { (one, other) -> one == other }
+                val canRestoreDefaults = mappedManualExclusionList.isNotEmpty()
+
+                canProtect to canRestoreDefaults
+            }
+            .flowOn(Dispatchers.IO)
```

Comment:
Internal activity so we are not as tidy as for production code, eg. we're not big in using view models, we don't bother using the `DispatcherProvider` for better testing